src/xrCore/xrCore.h

```diff
@@ -102,7 +102,7 @@
 #endif
 
 #include <time.h>
-#define MODULE_NAME "xrCore.dll"
+#define MODULE_NAME "xrCore"
```

Comment:
The only place it's used is xrCore.cpp:49:
```cpp
GetModuleFileName(GetModuleHandle(MODULE_NAME), fn, sizeof(fn));
```
So actually `MODULE_NAME` should be either defined in xrCore.cpp or inlined on the line above.